plugins/importexport/native/NativeImportExportPlugin.inc.php

```diff
@@ -249,9 +258,18 @@ function exportIssues($issueIds, $context, $user) {
 			$issue = $issueDao->getById($issueId, $context->getId());
 			if ($issue) $issues[] = $issue;
 		}
-		$issueXml = $exportFilter->execute($issues);
-		if ($issueXml) $xml = $issueXml->saveXml();
-		else fatalError('Could not convert issues.');
+		libxml_use_internal_errors(true);
+		$issueXml = $exportFilter->execute($issues, true);
+		$xml = $issueXml->saveXml();
+		$errors = array_filter(libxml_get_errors(), create_function('$a', 'return $a->level == LIBXML_ERR_ERROR ||  $a->level == LIBXML_ERR_FATAL;'));
+		if (!empty($errors)) {
+			$charset = Config::getVar('i18n', 'client_charset');
+			header('Content-type: text/html; charset=' . $charset);
+			echo '<html><body>';
+			$this->displayXMLValidationErrors($errors, $xml);
+			fatalError(__('plugins.importexport.common.error.validation'));
+			echo '</body></html>';
+		}
```

Comment:
Same as above: extra space after `||` and then move the closing HTML above the `fatalError`.